tests/test-tokenizer-1-bpe.cpp

```diff
@@ -93,7 +93,17 @@ int main(int argc, char **argv) {
     }
     // TODO: why doesn't this work for the full range of Unicodes?
```

Comment:
Could be helpful to change the comment?

Reply:
Does `Restrict to unicode planes` sound good?